test/inductor/test_torchinductor.py

```diff
@@ -7513,6 +7513,26 @@ def fn(in_ptr0, in_ptr1, in_ptr2):
             ),
         )
 
+    def test_indirect_broadcast_embedding(self):
+        B, T, D, V = (8, 2048, 4096, 8192)
+        op = nn.Embedding(V, D).to(self.device).to(torch.float32)
+        shared_weight = op.weight.data
+        _input = torch.randint(0, V, (B, T), device=self.device)
+        self.common(op, (_input,))
+        code = run_and_get_triton_code(
+            torch.compile(op),
+            _input,
+        )
+        # Check if the indices tensor are accessed via x dimension and the tiling works
+        for string in [
+            "xnumel = 16384",
+            "xoffset = tl.program_id(0) * XBLOCK",
+            "xindex = xoffset + tl.arange(0, XBLOCK)[:, None]",
+            "x1 = xindex",
+            "tmp0 = tl.load(in_ptr0 + (x1), None,",
+        ]:
+            self.assertTrue(string, code)
```

Comment:
Your test should always pass since 'string' is 'True'

Reply:
Thanks for catching it! will fix it in next commit!